### PR TITLE
PartDesign: fix task dressup/transformed creation cancel

### PR DESCRIPTION
**Pull request by realthunder**
https://forum.freecadweb.org/viewtopic.php?f=18&t=43578&sid=f3301cd4d06504841094f9e7af3099b4#p371652

### CODE REVIEW

File: src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp [hunk 1]
```diff
@@ -79,11 +82,17 @@ TaskDressUpParameters::~TaskDressUpParameters()
...
     int tid = 0;
     const char *name = App::GetApplication().getActiveTransaction(&tid);
+    if(tid && tid == transactionID)
+        return;
+
     std::string n("Edit ");
-    n += DressUpView->getObject()->Label.getValue();
-    if(!name || n != name)
+    n += DressUpView->getObject()->getNameInDocument();
+    if(!name || n!=name)
```
Comment (donovaly) [inline src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp]: Why this change? I think the style guideline is to have a space around any operator. I see this style is used everywhere in the code thus we should keep it.

Comment (realthunder) [reply src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp]: It is copied from my fork. Where is the guideline you referred to? I'll have a look. It's kind of hard to change the habit, if you write code as much as I do.

Comment (donovaly) [reply src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp]: I already asked for an official style guideline. The answer was that there is no such document and I should just follow the style by the existing documents. 

Here you changed the existing style.

Personally, I also use spaces around operators as done in most of FC's code. It improves the readability in my opinion.

Comment (realthunder) [reply src/Mod/PartDesign/Gui/TaskDressUpParameters.cpp]: Changed back now.

**Comment by wwmayer** [comment]
Merged.
